src/pages/privacy-policy.page.tsx

```diff
@@ -25,40 +29,65 @@ interface PrivacyPolicyPageProps {
 }
 
 export default function PrivacyPolicyPage({ post }): JSX.Element {
+  const { t } = useTranslation("page-privacypolicy");
+  const contentRef = useRef<HTMLDivElement>(null);
+
   return (
     <>
-      <Head title={post.title} description={post.description} />
-      <PageHeader title={post.title}>
-        <div className="mt-10 flex flex-wrap">
+      <div className="py-8 md:py-8 lg:py-16 border-b border-gray-800 border-opacity-50">
+        <Container className="flex flex-row gap-x-12 relative">
+          <Head title={post.title} />
```

Comment:
actually this is to add `<meta>` to the page, we can place it outside `<Container>`
```suggestion
          <Head title={Header.subtitle} />
```

Reply:
`post.title` will return `DeFiChain's Privacy Policy`, instead of `Header.subtitle`

Reply:
subtitle is `Privacy Policy` right? I chose that so that the browser tab's title is clear and concise compared to title
<img width="281" alt="image" src="https://user-images.githubusercontent.com/40191153/231405213-4db265af-d596-4933-8821-2cb738ee92a6.png">

Reply:
@chloezxyy and this change `<Head title={Header.subtitle} />` this is to taking the title from your `public/locales/en-US/page-privacypolicy.json`

Also can we place it as the immediate first child after `<>`, like i said this is adding `<meta>` so it doesnt affect the UI layout?